router/fastdp.go

```diff
@@ -856,7 +856,9 @@ func (fwd *fastDatapathForwarder) handleCryptoInitSARemote(msg []byte) {
 		return
 	}
 
-	if !fwd.isOutboundIPSecEstablished {
+	// FastDatapathCryptoInitSARemote can be received before Confirm'ing
+	// connection, thus before InitSALocal.
+	if fwd.confirmed && !fwd.isOutboundIPSecEstablished {
```

Comment:
`fwd.confirmed` denotes that `Confirm` has been successfully called, and `Confirm` ensures the initialization of `heatbeatTimer`.